ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/MainnetProtocolSpecs.java

```diff
@@ -193,6 +193,12 @@ private MainnetProtocolSpecs() {}
         .name("Constantinople");
   }
 
+  public static ProtocolSpecBuilder<Void> privacyDefinition(final int chainId) {
```

Comment:
As per comments about how to build privacy into the protocol schedule, don't think privacy should be defined as a spec built on top of constantinople, rather something that can be applied to any spec during building.